src/Debug.c

```diff
@@ -14,8 +14,14 @@
 
 #define FORMATTED_TIME_SIZE  1024
 
-int ros_DebugSocket = -1;
-struct sockaddr_in ros_debug_destAddr1;
+typedef struct
+{
+    UINT8 enabledPortCount;
+    int debugSocket[MAX_NETWORK_PORTS];
+    struct sockaddr_in destAddr[MAX_NETWORK_PORTS];
+} userLanInfo;
+
+userLanInfo debugPorts = {0};
```

Comment:
```suggestion
userLanInfo ros_debugPorts = {0};
```

This all exists in global kernel space. Easy to conflict with other global names.